### PR TITLE
Update vagrant file to allow for HL app to be run with it #864

### PR DESCRIPTION
**Pull request by RobStallion**
Update vagrant file to allow for HL app to be run with it #864 
Update README getting started instructions #864 #842

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,43 +6,40 @@ opportunities to improve communication between service users, carers and
...
+
+## Using the virtual machine
+
+You will need to be given access to this VM before you can perform the following steps:
+
+* `ssh hladmin@51.140.86.5`
+* `sudo -i`
+* `source /home/hladmin/.profile`
+* `cd /home/hladmin/hl-app`
+* `mix phoenix.server`
```
Comment (nelsonic) [inline README.md]: we won't be running the app this way. 
I've granted you access to the staging machine in order to work on the ePJS integration.
Apologies for the confusion. but we _cannot_ allow developers to login to the Staging server after the app goes live on Azure.

File: Vagrantfile [hunk 2]
```diff
@@ -6,36 +6,35 @@ sudo -i
...
+apt-get -y install g++ git git-core nodejs npm
+npm install n -g
+n stable
 node -v
 
 # PostgreSQL
-sudo apt-get install postgresql postgresql-contrib
-psql -u postgres postgres
+apt-get install postgresql postgresql-contrib -y
+sudo -u postgres psql -c "ALTER USER postgres PASSWORD 'postgres';"
```
Comment (nelsonic) [inline Vagrantfile]: if we have `sudo -i` above (_line 4_) we _should_ no longer need to call each command with `sudo` ... right?

File: README.md (hunk 1 above)
Comment (RobStallion) [inline README.md]: Oh my mistake. Thanks for clearing that up. Will update that right now 👍

File: Vagrantfile (hunk 2 above)
Comment (RobStallion) [inline Vagrantfile]: I think that this changes the Ubuntu user role from root to postgres. I couldn't seem to get it to work without it.

Comment (nelsonic) [inline Vagrantfile]: @RobStallion does that mean all subsequent commands are run as the `postgres` user?

**Pull request merged**